config/initializers/figaro.rb

```diff
@@ -12,6 +12,8 @@
   'MHV_SM_APP_TOKEN',
   'EVSS_BASE_URL',
   'MVI_URL',
+  'MVI_OPEN_TIMEOUT',
```

Comment:
Would it be possible to just default these values to some values in the event the environment variables dont exist?  It seems a little harsh to require them in Figaro.